docs/Guide/preconditions/channel-types.mdx

```diff
@@ -3,4 +3,78 @@ sidebar_position: 3
 title: Setting the types of channel a command can run in
 ---
 
-## TODO
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The [`runIn`][runin] command option can be used to specify the types of channels a command can run in. This can be
+useful if you're developing a command that, for example, displays the roles of a user. In that scenario, you'll want to
+make sure that the command can only be run in guild channels.
+
+:::info
+
+[Valid values][runintypes] for `runIn` are `'DM'`, `'GUILD_TEXT'`, `'GUILD_NEWS'`, `'GUILD_NEWS_THREAD'`,
```

Comment:
Please don't enumerate the types here! In the event we modify them, we have to remember to update it here too. Just mention where you can see the valid types instead